src/dali/plugin/input/csv/nexo.py

```diff
@@ -0,0 +1,150 @@
+            # read CSV with header and skip first row
+            lines = reader(transaction_csv_file)
+
+            # Skip header line
+            header = next(lines)
+            self.__logger.debug("Header: %s", header)
+
+            for line in lines:
+                raw_data: str = self.__DELIMITER.join(line)
+                self.__logger.debug("Transaction: %s", raw_data)
+
+                transaction_id: str = line[self.__TRANSACTION_ID_INDEX].strip()
+                transaction_type: str = line[self.__TRANSACTION_TYPE_INDEX].strip()
+                currency: str = line[self.__CURRENCY_INDEX].strip()
+                amount = line[self.__AMOUNT_INDEX].strip()
+                timestamp_with_timezone = f"{line[self.__TIMESTAMP_INDEX].strip()} -00:00"
+
+                if transaction_type in [_INTEREST, _FIXED_TERM_INTEREST]:
+                    result.append(
+                        InTransaction(
+                            plugin=self.__NEXO,
+                            unique_id=transaction_id,
+                            raw_data=raw_data,
+                            exchange=self.__NEXO,
+                            holder=self.account_holder,
+                            timestamp=timestamp_with_timezone,
+                            asset=currency,
+
+                            transaction_type=Keyword.INTEREST.name,
+                            # nexo does give us the spot price, but it's often 0 if subcent
+                            spot_price=Keyword.UNKNOWN.value,
+                            crypto_in=amount,
+                            fiat_fee="0",
+                        )
+                    )
+                elif transaction_type in [_LOCKING_TERM_DEPOSIT, _UNLOCKING_TERM_DEPOSIT]:
+                    # I don't think we need to record locking/unlocking deposits for term interest
+                    self.__logger.debug("Skipping lock or unlock deposit: %s", line)
+                elif transaction_type == _DEPOSIT:
+                    result.append(
+                        IntraTransaction(
+                            plugin=self.__NEXO,
+                            unique_id=transaction_id,
+                            raw_data=raw_data,
+                            timestamp=timestamp_with_timezone,
+                            asset=currency,
+
+                            crypto_received=amount,
+
+                            # most likely, it's you, but we can't say for sure
+                            from_exchange=Keyword.UNKNOWN.value,
+                            from_holder=Keyword.UNKNOWN.value,
+
+                            to_exchange=self.__NEXO,
+                            to_holder=self.account_holder,
+
+                            # we do know the spot price, but nexo seems to do some aggressive rounding
+                            spot_price=Keyword.UNKNOWN.value,
+                            crypto_sent=Keyword.UNKNOWN.value,
```

Comment:
@eprbell this line actually caused me issues with `rp2`. Should I use `0.0` here instead? That seemed to solve the issue when I was trying to get rp2 to generate.

Reply:
What was the issue: could you paste a stack trace?

Reply:
Here's the error message on the rp2 side:

```
Exception: Encountered an unresolved DaLI transaction (read DaLI's documentation / FAQ to learn how to resolve this issue): 
```

When I filled in the `crypto_sent` value with `0.0` the issue was fixed. Any ideas here? In this case we are just receiving crypto, so it makes sense that the sent would be zero?